example.config.json

```
@@ -2,24 +2,25 @@
     "prover_name_prefix": "sindri_",
     "keys_dir": "keys",
     "coordinator": {
-        "base_url": "https://your-coordinator-api",
+        "base_url": "https://coordinator-ap:80",
         "retry_count": 3,
         "retry_wait_time_sec": 5,
         "connection_timeout_sec": 60
     },
     "l2geth": {
-        "endpoint": "https://your-l2-rpc"
+        "endpoint": "https://l2-rpc:8545"
     },
     "prover": {
         "circuit_type": 3,
         "circuit_version": "v0.13.1",
         "n_workers": 1,
         "cloud": {
-            "base_url": "https://sindri.app",
+            "base_url": "https://sindri.app/api/v1/",
```

This revision won't work. It was purposely changed in https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/15

I suspect you are working from an older version of main and need to pull in the latest remote changes
```suggestion
            "base_url": "https://sindri.app",
```

---

Done:  [1845dc2](https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/20/commits/1845dc24e37aa5c342d80b0165806f5bbd95d423)